### DISCUSSION TITLE
Errors logged by WalletConnect v3 are truncated (vs full error message expected)

### DISCUSSION THREAD

**Discussion by SergeyVolynkin** [WalletConnect/walletconnect-monorepo#3833] [Developer Support] [answered]
Hi WalletConnect team,

We monitor our app in Senry(sentry.io) –– and we receive errors thrown by WalletConnect in truncated format, do you have some code that intentionally restricts error length on WalletConnect-side?

```json5
// Versions used:
"@web3modal/wagmi": "^3.1.0",
"viem": "^1.16.6",
"wagmi": "^1.4.5",
"@sentry/react": "^7.72.0",
```

Here is an example:
error ends with `Details: Insuff...` creates some confusion when debugging. It seems that the error itself is logged like that and not Sentry's fault 🤔.
![Screenshot 2023-10-27 at 1 20 02 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/34238697/52693d07-a5cb-44db-bf26-753e10b8cfe1)

**Comment by chris13524** [comment, marked as answer]
Hey @SergeyVolynkin our blockchain API is [open source](https://github.com/walletconnect/blockchain-api) and the text "Insuff" isn't in our codebase anywhere. We also do not truncate error responses, at least intentionally. However, this may be an error coming from one of the upstream RPC providers for this chain, which we pass-through any response status & body and do not have control over this.

**Reply by SergeyVolynkin** [reply]
Thank you @chris13524!
